.circleci/config.yml

```diff
@@ -113,6 +113,84 @@ jobs:
             package_cloud push opennms/plugin-snapshot/el/7 assembly/opennms-rpm/target/rpm/opennms-oce-plugin/RPMS/noarch/*
             package_cloud push opennms/plugin-snapshot/el/7 assembly/sentinel-rpm/target/rpm/sentinel-oce-plugin/RPMS/noarch/*
 
+  smoke-test:
+    machine: true
```

Comment:
Can we get this to work in a container instead of using the machine image?

Reply:
I think it's possible, but I need to look into it further to see how to properly get docker-in-docker to work on circleci. You can bind mount like this: `docker run -v /var/run/docker.sock:/var/run/docker.sock` and that should make it work once we add docker to our build-env image. However I'm not sure if/how we can control how our docker image is started on circleci.

The alternative is to keep using the machine image but have to make some somewhat ugly changes to how we cache the maven repo since the machine image does not have write access to /root. I've went with this approach for now.